golem-worker-service-base/src/gateway_execution/gateway_binding_resolver.rs

```diff
@@ -12,553 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::gateway_api_definition::http::{CompiledHttpApiDefinition, VarInfo};
```

Comment:
Most of the logic here was simplified and inlined into the executor. That way we also do not need to convert requests to json multiple times (and can do it differently for different binding types)